cvr/src/main/java/org/cbioportal/cmo/pipelines/cvr/mutation/CVRMutationDataReader.java

```diff
@@ -175,16 +175,16 @@ public void handleLine(String line) {
         }
         reader.close();
         log.info("Loaded " + String.valueOf(recordsToAnnotate.size()) + " records from MAF");
-        annotateRecordsWithPOST(recordsToAnnotate);
+        annotateRecordsWithPOST(recordsToAnnotate, forceAnnotation);
```

Comment:
https://github.com/knowledgesystems/cmo-pipelines/pull/652/files#diff-7258385239600627f9ec5efeef299ee3L193

Reply:
Added this argument `reannotate` back in to differentiate between whether we need to "force annotation".

For records read from JSON we only want to annotate "unannotated records" unless we "force annotation". This original logic was lost -- refer to link above to see what was dropped.